app/models/cart.rb

```diff
@@ -18,6 +18,16 @@ def count_of(item_id)
     contents[item_id.to_s]
   end
 
+  def update_quantity(params)
+    self.contents.select { |item, _quantity| item == params[:id] }
+                    .map { |item, _quantity|
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 16) spaces for indenting an expression spanning multiple lines.